models/jules/R/write.config.JULES.R

```diff
@@ -554,13 +556,19 @@ write.config.JULES <- function(defaults, trait.values, settings, run.id) {
 
   ## set npft to the value needed for surface type definition
   npft <- max(c(npft, 5))
+  
+  ## Edit jules_surface_types.nml to set correct number of PFTS
 
   ## --------------------------- END PFTS ------------------------------------------
-
-  ## Edit jules_surface_types.nml to set correct number of PFTS
+  
+  ## For JULES and, Enter defaults for IC.DAT
 
   ## Edit INITIAL_CONDITIONS.NML soil carbon LAI
+  
+  
   if(useTRIFFID){
+    
+    PEcAn.logger::logger.warning("No Initial Conditions, using defaults")
```

Comment:
Why is there a warning about no initial conditions BEFORE the check to see if there are initial conditions?